src/rgw/rgw_user.cc

```diff
@@ -1735,6 +1735,7 @@ int RGWUser::init(RGWUserAdminOpState& op_state)
   }
 
   if (!uid.empty() && (uid.compare(RGW_USER_ANON_ID) != 0)) {
+    op_state.set_reserved_user(false);
```

Comment:
As `RGWUserAdminOpState` 's default user is anonymous one(see https://github.com/ceph/ceph/blob/master/src/rgw/rgw_user.h#L478 ), so  `reserved_user` is true by default.